CONTRIBUTING.md

```diff
@@ -177,18 +177,31 @@ a new application. The name of the application should consist of the
 component name in lowercase, with words separated by dashes (`-`). The
 deployment server will provide a remote repository URL that will
 contain the app. Add it to the list of remotes by running `git
-remote add gallery-test [deployment server git URL]`.
+remote add [app name]-test [deployment server git URL]`.
 
-#### Step 2: Edit and commit the `Procfile`
+#### Step 2: Edit and commit app-specific files
+
+##### Step 2a: Edit the `Procfile`
 Edit the `Procfile` in the repository to say `gunicorn
-app_name:server`, where `app_name` is the name of the app you want to
-deploy in the `tests/dashbio_demos/` folder. Then, commit the change
-you made to the `Procfile`.
+tests.dashbio_demos.app_name:server`, where `app_name` is the name of
+the app you want to deploy in the `tests/dashbio_demos/` folder. 
+
+##### Step 2b: Edit `config.py`
+Edit the `config.py` file in the repository such that the variable
+`DASH_APP_NAME` is set to the name of your app, but with `app`
```

Comment:
```suggestion
`DASH_APP_NAME` be set to the name of your app, but with `app`
```

Reply:
@mkcor This seems grammatically fine to me -- not sure what the issue is

Reply:
Just a suggestion... I thought the subjunctive mood would be more accurate since we're expressing a goal, a desire, not a factual reality (here's a discussion, for example: https://english.stackexchange.com/questions/58634/so-that-it-be-vs-so-that-it-is).

Reply:
Ah, that makes sense -- thanks! Learned something today :)